source/tls_channel_handler.c

```diff
@@ -244,21 +238,19 @@ int aws_tls_ctx_options_init_client_mtls_with_pkcs11(
             aws_string_new_from_cursor(allocator, &pkcs11_options->private_key_object_label);
     }
 
-    struct aws_pkcs11_tls_op_handler *pkcs11_handler = aws_pkcs11_tls_op_handler_new(
+    struct aws_custom_key_op_handler *pkcs11_handler = aws_pkcs11_tls_op_handler_new(
         allocator,
         pkcs_lib,
         pkcs_user_pin,
         pkcs_token_label,
         pkcs_private_key_object_label,
-        pkcs_has_slot_id ? &pkcs_slot_id : NULL,
-        pkcs11_options->cert_file_path,
-        pkcs11_options->cert_file_contents);
+        pkcs_has_slot_id ? &pkcs_slot_id : NULL);
 
     if (pkcs11_handler == NULL) {
         goto error;
     }
 
-    // CLEANUP
+    /* CLEANUP */
     if (pkcs_lib != NULL) {
         aws_pkcs11_lib_release(pkcs_lib);
```

Comment:
trivial/style: don't have aws_tls_ctx_options_init_client_mtls_with_pkcs11() function do acquire()/release() on the pkcs11_lib. It's going to be "held" by the aws_pkcs11_tls_op_handler()